portfolio/src/main/webapp/index.html

```diff
@@ -31,16 +31,21 @@ <h1>AnaK's Portfolio</h1>
       <div id="random-image-container"></div>
 
       <p>Click <a href="placesGallery.html">here</a> to see some of my favourite places!</p>
-
+      
       <h1>Comments:</h1>
       <ul id="comments-history"></ul>
       
+      <p>Choose the maximum number of comments to be displayed:</p>
+      <input type="number" id="commentsLimit" name="commentsLimit" min="0" max = "100" onchange="getComments()" >
+      
       <form action="/data" method="POST">
       <p>Write a new comment:</p>
       <textarea name="new-comment"></textarea>
 
       <input type="submit" />
       </form>
+      
+      <button onclick="deleteAllComments()">Delete all the comments</button>
```

Comment:
This PR seems to do 3 things:
- Limit number of comments (as you mention in the description).
- Add functionality to delete all comments.
- Changes the storage backend from in-memory to datastore.

Generally that's an antipattern, each PR should do one thing and one thing well. git is actually pretty good at merging changes and creating multiple PRs is actually a great way to get the reviewed faster (cause they're much easier to understand), even if it intuitively seems to slow you down.

Here's a good read:
https://essenceofcode.com/2019/10/29/the-art-of-small-pull-requests/

We also have internal best practices about this (go/small-cls), but not sure if you have access to this.

I cannot emphasize enough how useful it is to get into this mindset of splitting changes up. Up to you in this case whether you want to try to break it up or not.

Do not be afraid of having multiple PRs open at the same time, otherwise you're deliberately slowing you down. This is the kind of problem that git is meant to solve.